packages/backend/src/plugins/auth.ts

```diff
@@ -24,5 +28,28 @@ export default async function createPlugin({
   config,
   discovery,
 }: PluginEnvironment): Promise<Router> {
-  return await createRouter({ logger, config, database, discovery });
+  return await createRouter({
+    logger,
+    config,
+    database,
+    discovery,
+    providerFactories: {
+      ...defaultAuthProviderFactories,
```

Comment:
@Rugvip This looks unnecessary to me here, since the `defaultAuthProviderFactories` is later used inside [`createRouter`](https://github.com/backstage/backstage/blob/3b2d4d421ace5511a8be94e235ec97c708bfc564/plugins/auth-backend/src/service/router.ts#L84-L87) eventually. WDYT?